### PR TITLE
Some updates

### PR DESCRIPTION
**Pull request by Blacksmoke16**
* Replaces `assert` with https://github.com/athena-framework/validator
* Updates owner/shard/repo names for some shards
* Removes inactive shards

### CODE REVIEW

**Comment by veelenga** [comment]
@Blacksmoke16 thanks, i am tagging repo owners if they don't have plans/objections to remove their inactive shards.

---

FYI these items are going to be removed from the awesome-list since they started to break the [Contribution guidelines](https://github.com/veelenga/awesome-crystal/blob/master/.github/CONTRIBUTING.md). We consider these shards as not maintained or they don't have the periodical builds accessible on CI. 

If you would like to keep the link in the list, please follow the guidelines and let us know. If not, please just ignore it. Thanks. 

@j8r - https://github.com/j8r/clicr
@j8r - https://github.com/j8r/cride
@aca-labs - https://github.com/aca-labs/terminimal
@rodrigopinto - https://github.com/rodrigopinto/can_use
@blocknotes - https://github.com/blocknotes/magickwand-crystal
@naqvis - https://github.com/naqvis/containerregistry.cr
@SuperPaintman - https://github.com/SuperPaintman/ulid
@bmulvihill - https://github.com/bmulvihill/dispatch
@sam0x17 - https://github.com/sam0x17/gcf.cr
@Nephos - https://github.com/Nephos/wikicr

File: README.md [hunk 1]
```diff
@@ -235,7 +230,6 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [try.cr](https://github.com/maiha/try.cr) - Try monad
 
 ## Feature Flipping
- * [can_use](https://github.com/rodrigopinto/can_use) - It is a minimalist feature toggle/flag for crystal based on yaml
```
Comment (veelenga) [inline README.md:238]: the section can be removed

**Comment by j8r** [comment]
Fine for me to remove clicr. However, I think Cride is quite unique and could be useful to be kept - that's up to you.

**Comment by Blacksmoke16** [comment]
@j8r Would you mind setting up a periodic build for it?

**Comment by j8r** [comment]
Done, but it doesn't guaranty that I will see it failing :sweat_smile:

File: README.md (hunk 1 above)
Comment (rodrigopinto) [reply README.md:238]: can_use was built for personal use, not intention it becomes used by a large audience, but would be better to keep the "Feature Toggle" section at least to promote other solutions like https://github.com/metaware/flipper?

**Comment by Nephos** [comment]
Yes we may remove wikicr since I don't finish it. I will probably do an update for crystal v1 but not before I think.

**Comment by j8r** [comment]
@Blacksmoke16 I'm going to update Clicr (got an idea to replace macros by `forall` fu).
Of course, an monthly build is going to be added.

**Comment by j8r** [comment]
@Blacksmoke16 I'm going to update Clicr (got an idea to replace macros by `forall` fu).
Of course, a monthly build is going to be added.

File: README.md (hunk 1 above)
Comment (veelenga) [reply README.md:238]: Flipper's latest commit was done 2 years ago. I would still suggest removing the section completely.

**Comment by veelenga** [comment]
@Blacksmoke16 i'm ok merging this once comments addressed and conflicts are resolved. Thanks for the help

**Comment by Blacksmoke16** [comment]
@veelenga :+1:, I'll get those patched up today.

File: README.md [hunk 2]
```diff
@@ -116,12 +116,10 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [x_do.cr](https://github.com/woodruffw/x_do.cr) - Bindings for libxdo ([`xdotool`](https://github.com/jordansissel/xdotool))
 
 ## Caching
- * [bloom_filter](https://github.com/crystal-community/bloom_filter) - Implementation of Bloom filter
  * [crystal-memcached](https://github.com/comandeo/crystal-memcached) - Implementation of a memcached client
 
 ## CLI Builders
  * [admiral](https://github.com/jwaldrip/admiral.cr) - A robust DSL for writing command line interfaces
- * [clicr](https://github.com/j8r/clicr) -  A simple declarative command line interface builder
```
Comment (j8r) [inline README.md:124]: Can be kept, a CRON is added now

File: README.md [hunk 3]
```diff
@@ -130,13 +128,11 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 
 ## CLI Utils
  * [climate](https://github.com/Sija/climate.cr) - Tiny tool to make your CLI output 🌈  coloured
- * [cride](https://github.com/j8r/cride) - A light CLI text editor/IDE
```
Comment (j8r) [inline README.md:133]: Can be kept, a CRON is added now.

File: README.md [hunk 4]
```diff
@@ -338,15 +326,13 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
...
  * [immutable](https://github.com/lucaong/immutable) - Implementation of thread-safe, persistent, immutable collections
  * [iterm2](https://github.com/toddsundsted/iterm2) - Display images within the terminal using the ITerm2 Inline Images Protocol
- * [lirith](https://github.com/lirith-engine/lirith) - Graphics engine
  * [message_verifier.cr](https://github.com/danielwestendorf/message_verifier.cr) - Verify and generate Rails `ActiveSupport::MessageVerifier` signed tokens
  * [monads](https://github.com/alex-lairan/monads) - Monad implementation
  * [pinger](https://github.com/spider-gazelle/pinger) - Ping IP addresses and DNS entries without requiring sudo
  * [port_midi](https://github.com/jimm/crystal_port_midi) - Crystal C bindings for the PortMIDI cross-platform MIDI I/O library
  * [retriable.cr](https://github.com/Sija/retriable.cr) - Simple DSL to retry failed code blocks
  * [simple_retry](https://github.com/spider-gazelle/simple_retry) - Simple tool for retrying failed code blocks
  * [sslscan.cr](https://github.com/NeuraLegion/sslscan.cr) - Crystal shard wrapping the rbsec/sslscan utility
```
Comment (Blacksmoke16) [inline README.md:335]: @Sija Just as a heads up the CI for this is actually under `Sija` not `NeuraLegion`, so the script used to handle these updates isn't picking it up correctly.  Could you get that corrected so it doesn't mistakenly get removed next cleanup?

**Review by veelenga — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (Sija) [reply README.md:335]: Will do, thanks for the heads up! 🙏

**Pull request merged**